### PR TITLE
Splitting up the views

### PR DESCRIPTION
**Pull request by mamazu**
Splitting up the views.

### CODE REVIEW

File: src/Factory/Cart/EstimatedShippingCostViewFactoryInterface.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * Created by PhpStorm.
+ * User: mamazu
+ * Date: 3/31/19
+ * Time: 7:01 PM
+ */
```
Comment (GSadee) [inline src/Factory/Cart/EstimatedShippingCostViewFactoryInterface.php]: Not needed docblock

File: src/Resources/config/app/config.yml [hunk 2]
```diff
@@ -1,9 +1,12 @@
 jms_serializer:
     metadata:
         directories:
-            sylius-shop-api:
-                namespace_prefix: "Sylius\\ShopApiPlugin\\View"
-                path: "@ShopApiPlugin/Resources/config/serializer"
+            sylius-shop-api_taxon:
```
Comment (GSadee) [inline src/Resources/config/app/config.yml]: `sylius-shop-api-taxon`?

**Comment by lchrusciel** [comment]
Rebase needed + can you also apply @GSadee comment?

**Comment by GSadee** [comment]
Thanks, @mamazu! :1st_place_medal:

**Pull request merged**